jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/impl/SignalDescImplTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.kie.services.impl;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+import org.jbpm.bpmn2.core.Message;
+import org.jbpm.bpmn2.core.Signal;
+import org.jbpm.kie.services.impl.model.SignalDescImpl;
+import org.jbpm.services.api.model.SignalDesc;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class SignalDescImplTest {
+
+    @Test
+    public void testEqualsHashCode() {
+        SignalDesc signal1 = SignalDescImpl.from(new Signal("pepe", "pepe", "pepe"));
+        SignalDesc signal2 = SignalDescImpl.from(new Signal("pepe", "pepe", "pepe"));
```

Comment:
ok, I technically "abuse" pepe here, but because I want to emphasize that everything is equals and despite of it, since one SignalDesc come from Signal object and another SignalDesc come from Message, theere are two instances at the end in the set. Changed anyway